src/Sidebar/SidebarTabs.test.js

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+import { render } from "@testing-library/react";
+import SidebarTabs from "./SidebarTabs";
+import userEvent from "@testing-library/user-event";
+
+describe("Sidebar tab tests", () => {
+  test("Should render SlidebarTab ", () => {
+    const activeTab = "manage_workflows";
+    const setOpenListTab = jest.fn();
+    const sampleLink = jest.fn();
+    const tabDetail2 = {
+      id: "samples",
+      title: "Samples",
+      icon: "code-fork",
+      link: sampleLink,
+    };
+
+    const { getByRole } = render(
+      <SidebarTabs
+        tabDetails={tabDetail2}
```

Comment:
Try to avoid naming component with magic numbers. 

```suggestion
        tabDetails={sampleTabDetail}
```